docs/content/en/docs/osmgmt/artifacts.md

```diff
@@ -997,6 +1042,25 @@ With `image-builder` versions `v0.2.1` and above (released with EKS-A version `v
    image-builder build --os <OS> --hypervisor <hypervisor> --release-channel <release channel> --<hypervisor>-config config.json --eksa-release v0.16.5
    ```
 
+#### Building images corresponding to dev versions of EKS-A
```

Comment:
im fine documenting this because i know we have users doing this, but we should probably add a warning or something that says this is not recommended or its an advanced use case or only do when directed by eks-a engineers, something